### PR TITLE
Math magicians: full website

### PR DESCRIPTION
**Pull request by FaranoshAmini**
In this pull request, I have implemented:

 -  Created a website consisting of 3 pages: Home, Calculator, and Quote
 -  Add my own styles to improve the look and feel
 -  For every page, I have created a route
 -  Used [React Router](https://reactrouter.com/) to make the routes work
 -  And this is what the project looks like:

<p align="center"></p>
<p align="center"></p>
<p align="center"></p>

### CODE REVIEW

File: src/Components/Home.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        from the comfort of their own homes, Udemy strives to help individuals enhance their
+        skills and realize their visions. While users are able to individually sign up for courses,
+        this particular landing page promotes Udemy’s business plan. It offers companies a platform
+        which they can train and educate their employees, providing them with extra capabilities and
+        on a range of topics. This can help businesses support their employees while working.
+      </p>
+    </main>
+  );
+}
+
```
Comment (fobadara) [inline src/Components/Home.js:21]: [REQUIRED] Kindly align the texts the left just like the wireframe

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @FaranoshAmini, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing

## Required Changes ♻️
_Check the comments under the review._
[REQUIRED] I noticed that you did not align your calculator to the right and a text to the left. Your general layout doesn't look like the wireframe. Kindly ensure it looks like it.
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/Home.js (hunk 1 above)
Comment (FaranoshAmini) [reply src/Components/Home.js:21]: I have implemented the changes

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @FaranoshAmini,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ You used Gitflow correctly 👍 
✅ Linter checks passed 👍 
✅ App works fine and routing was implemented correctly 👍 

## Required Changes ♻️
_Check the comments under the review._

Hi!, I strongly believe what the previous reviewer was asking was about the Calculator page, to have text on the left and the calculator app on the right just as in the wireframe.

- [ ] Currently, you are having just the calculator on the center of the calculator page. Kindly add the text on the left that says "Le do some math" so you match the wireframe. 👍 

**See the screenshot below on the wireframe**

image

**Your Design**
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by mwanawabangona — APPROVED** [review]
Hi @FaranoshAmini 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by FaranoshAmini** [comment]
@AmaduKamara, @fobadara, and @mwanawabangona Thanks for your kind review and feedback.

**Pull request merged**